sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -295,7 +295,7 @@ SUBROUTINE CALRH_GSD(P1,T1,Q1,RHB)
 
       DO J=JSTA,JEND
         DO I=1,IM
-
+         IF (T1(I,J) < SPVAL) THEN
```

Comment:
Should add checks P1 and Q1 in line 298.

Reply:
will do

Reply:
done